src/routes/safe/components/Apps/utils.ts

```diff
@@ -34,6 +34,12 @@ export const staticAppsList: Array<StaticAppInfo> = [
     disabled: false,
     networks: [ETHEREUM_NETWORK.MAINNET],
   },
+  // Aave v2
+  {
+    url: `https://app.aave.com/`,
```

Comment:
🤔

Reply:
I think it is related to the base branch. I need this or `fix/2048-sApp-section-dissapears` because of the collection bug.